test/static.test.js

```diff
@@ -435,6 +435,54 @@ t.test('not found responses can be customized with fastify.setNotFoundHandler()'
   })
 })
 
+t.test('serving disabled', t => {
+  t.plan(3)
+
+  const pluginOptions = {
+    root: path.join(__dirname, '/static'),
+    prefix: '/static/',
+    serve: false
```

Comment:
I think if you are not serving a directory, the `root` and `prefix` options is not needed at all.

Maybe this step can be skipped completely

https://github.com/fastify/fastify-static/blob/5423247a51e3524b9c733b676d9c59549a2b4483/index.js#L12

Reply:
For `prefix`: I agree, but `root` gives me the possibility to just pass a filename to `reply.sendFile()` instead of a absolute path.